tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -485,7 +485,8 @@ private void handleOnePath(String path) {
             htmlMessage.setLength(0);
 
             final String oldValue = (baselineFileUnresolved == null) ? null : baselineFileUnresolved.getWinningValue(path);
-            if (skipForLimitedSubmission(path, errorStatus, oldValue)) {
+            final VoteStatus voteStatus = userVoteStatus.getStatusForUsersOrganization(sourceFile, path, organization);
```

Comment:
I don't understand this. We should be clear that when we talk about Error / Missing / Provisional, we are talking *not* about provisional in the user's org, but instead **globally** Provisional. That is, the **winning** path doesn't have draft=unconfirmed or draft=provisional (independent of Org of the user).

That is just like Missing. That means in this context, globally missing, not just that the user's org hasn't voted on the item.

Reply:
I need to study that... VoteStatus can be org-specific or org-neutral? Avoid getStatusForUsersOrganization?

Reply:
Looks like VoteStatus is only intended for organization-specific status, so it shouldn't be used for this new purpose.

Note that VettingViewer.FileInfo.handleOnePath already gets VoteStatus by calling getStatusForUsersOrganization (not all Dashboard notifications are org-neutral) -- so VoteStatus.provisionalOrWorse seemed read-made for the new purpose.

@macchiati I need to understand the best way to get "globally Provisional. That is, the winning path doesn't have draft=unconfirmed or draft=provisional"

also it looks like currently the Dashboard "Provisional" notification IS org-specific -- is that in disagreement with what you're saying it should be?

The dash notification is `NotificationCategory.notApproved` and it's based on `userVoteStatus.getStatusForUsersOrganization(sourceFile, path, organization) == VoteStatus.provisionalOrWorse`

```
        private void recordLosingDisputedEtc(String path, VoteStatus voteStatus, MissingStatus missingStatus) {
            switch (voteStatus) {
            case losing:
                if (choices.contains(NotificationCategory.weLost)) {
                    problems.add(NotificationCategory.weLost);
                    vc.problemCounter.increment(NotificationCategory.weLost);
                }
                String usersValue = userVoteStatus.getWinningValueForUsersOrganization(sourceFile, path, organization);
                if (usersValue != null) {
                    usersValue = "Losing value: <" + TransliteratorUtilities.toHTML.transform(usersValue) + ">";
                    appendToMessage(usersValue, htmlMessage);
                }
                break;
            case disputed:
                if (choices.contains(NotificationCategory.hasDispute)) {
                    problems.add(NotificationCategory.hasDispute);
                    vc.problemCounter.increment(NotificationCategory.hasDispute);
                }
                break;
            case provisionalOrWorse:
                if (missingStatus == MissingStatus.PRESENT && choices.contains(NotificationCategory.notApproved)) {
                    problems.add(NotificationCategory.notApproved);
                    vc.problemCounter.increment(NotificationCategory.notApproved);
                }
                break;
            default:
            }
        }
    }
```